node/src/p2p.rs

```diff
@@ -899,6 +939,55 @@ where
             gossipsub::MessageAcceptance::Ignore
         }
     }
+
+    #[instrument(skip_all)]
+    async fn on_bad_encoding_fraud_sub_message(
+        &mut self,
+        data: &[u8],
+        peer: &PeerId,
+    ) -> gossipsub::MessageAcceptance {
+        let Ok(befp) = BadEncodingFraudProof::decode(data) else {
+            trace!("Malformed or invalid bad encoding fraud proof from {peer}");
```

Comment:
As I understand it, this branch is for malformed proof we couldn't parse, proof validity is checked later. (If we care to make that distinction, nitpicking here).

Reply:
```suggestion
            trace!("Malformed bad encoding fraud proof from {peer}");
```